Firestore/core/src/firebase/firestore/core/view_snapshot.h

```diff
@@ -177,6 +178,7 @@ class ViewSnapshot {
   }
 
   std::string ToString() const;
+  friend std::ostream& operator<<(std::ostream& out, const ViewSnapshot& value);
```

Comment:
I think you can drop the friend declaration.

Reply:
Dropping `friend` also means moving the declaration out of the class, which is annoying.